fe/fe-core/src/main/java/org/apache/doris/catalog/external/HMSExternalTable.java

```diff
@@ -456,6 +448,19 @@ private void initPartitionColumns(List<Column> schema) {
 
     @Override
     public Optional<ColumnStatistic> getColumnStatistic(String colName) {
+        makeSureInitialized();
+        switch (dlaType) {
+            case HIVE:
+                return getHiveColumnStats(colName);
+            case ICEBERG:
+                return StatisticsUtil.getIcebergColumnStats(colName, HiveMetaStoreClientHelper.getIcebergTable(this));
```

Comment:
We should unify the method of `HiveMetaStoreClientHelper.getIcebergTable` and `IcebergExternalCatalog.getIcebergTable()`

Reply:
And it is strange and if this is a hive table, call the method `getHiveColumnStats()`, which is a private method of this class. But if this is an iceberg table, call the static method `StatisticsUtil.getIcebergColumnStats`.
Why this 2 methods not in same level.

Reply:
The reason of these two issues is we have 2 implementations for Iceberg table. One is in HMSExternalTable, the other is IcebergExternalTable. They are using different catalog in the getIcebergTable() function.
I put getIcebergColumnStats in StatisticsUtil as a static function so that both HMSExternalTable and IcebergExternalTable could use it. But for hive table, it is only used in HMSExternalTable.
I think to solve the problem, we should refactor the code, use a single IcebergTable to handle all different Catalogs (hms, glue, dlf...).